examples/panel/stock-market-chatbot/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11
```

Comment:
we don't need the dockerfile, right? we can just deploy it as a panel app

Reply:
Resolved https://github.com/ploomber/doc/pull/124/commits/fdda92e7f40bb1478e9368e1e0e24e7605ef4fd2